scikit_mol/transformers.py

```diff
@@ -43,9 +43,9 @@ def transform(self, X, y=None):
 
 
 class MACCSTransformer(FpsTransformer):
-    def __init__(self):
-        self.nBits = 167
-        pass
+    def __init__(self, nBits=167):
```

Comment:
We should not have a configurable parameter in the __init__ function that will make the object fail

Reply:
yes I agree on that, I was not sure if there was any way to configure it but if we cannot configure it then it should not be allowed as an argument